content/committees/specification/guide/index.md

```diff
@@ -0,0 +1,80 @@
+3. Send an email to the [Specification Committee](mailto:jakarta.ee-spec@eclipse.org) to request a creation review.
+
+(*) It may be a good idea to socialize the idea with the Specification Committee to solicit support before proceeding with the project creation.
+
+## Plan Review {#plan}
+
+![Plan Review](JESP_plan-review.png)
+
+* Each specification project needs to engage in a Plan Review prior to any extensive development effort in support of a new Specification Version. 
+
+### Steps Involved
+
+1. Submit a Pull Request to the [Jakarta EE Specifications repository](https://github.com/jakartaee/specifications) using the **[plan review template](https://github.com/jakartaee/specifications/blob/master/plan_review_pr_template.md)**.
+2. Send an email to the [Specification Committee](mailto:jakarta.ee-spec@eclipse.org) to request a plan review.
+
+## Progress Review {#progress}
+
+![Progress Review](JESP_progress-review.png)
+
+* A progress review can be initiated by the project when they want to inform the specification committee about their progress, but are not ready for a release yet.
+* A progress review may be requested by the specification committee if no progress has been made in 12 months.
+
+### Steps Involved
+
+1. Submit a Pull Request to the [Jakarta EE Specifications repository](https://github.com/jakartaee/specifications) using the **[progress review template](https://github.com/jakartaee/specifications/blob/master/progress_review_pr_template.md)**.
+2. Send an email to the [Specification Committee](mailto:jakarta.ee-spec@eclipse.org) to request a plan review.
+
+## Release Review {#release}
+
+![Release Review](JESP_release-review.png)
+
+* A final version of a specification cannot be made generally available until after engaging in a successful release review (with corresponding super-majority approval from the specification committee).
+
+### Steps Involved
+
+1. Submit a Pull Request to the [Jakarta EE Specifications repository](https://github.com/jakartaee/specifications) using the **[release review template](https://github.com/jakartaee/specifications/blob/master/pull_request_template.md)**.
+2. Send an email to the [Specification Committee](mailto:jakarta.ee-spec@eclipse.org) to request a release review.
+
+## Service Release {#service}
+
+* There is no formal release review required for a Specification service release (x.y.z) as long as the JESP definition of a “service release” is adhered to.
+* That means that no functionality changes or increase in IP scope are permitted in a service release. 
+
+### Steps Involved
+
+1. Submit a Pull Request to the [Jakarta EE Specifications repository](https://github.com/jakartaee/specifications) using the **[service release template](https://github.com/jakartaee/specifications/blob/master/service_release_pr_template.md)**.
+2. Send an email to the [Specification Committee](mailto:jakarta.ee-spec@eclipse.org) to announce that the PR is ready for review.
```

Comment:
On this one, the 3rd step would be something like, "3. Wait for the PR approval and merge before proceeding."?